<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frameMenu">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="btnImagenes">
         <property name="font">
          <font>
           <family>Copperplate Gothic Bold</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0, 255);</string>
         </property>
         <property name="text">
          <string>üì∑ Imagenes</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnSenales">
         <property name="font">
          <font>
           <family>Copperplate Gothic Bold</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 170, 255);</string>
         </property>
         <property name="text">
          <string>üìàSe√±ales</string>
         </property>
         <property name="style" stdset="0">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnCSV">
         <property name="font">
          <font>
           <family>Copperplate Gothic Bold</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0, 255);</string>
         </property>
         <property name="text">
          <string>üìÅ CSV</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnHistorial">
         <property name="font">
          <font>
           <family>Copperplate Gothic Bold</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0, 255);</string>
         </property>
         <property name="text">
          <string>üïò Historial</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnCerrarSesion">
         <property name="font">
          <font>
           <family>Copperplate Gothic Bold</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(85, 170, 255);</string>
         </property>
         <property name="text">
          <string>üö™ Cerrar Sesi√≥n</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frameContenido">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="labelTituloDashboard">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>10</y>
         <width>171</width>
         <height>81</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Copperplate Gothic Bold</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>CONTENIDO</string>
       </property>
      </widget>
      <widget class="QStackedWidget" name="stackprincipal">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>70</y>
         <width>301</width>
         <height>471</height>
        </rect>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page"/>
       <widget class="QWidget" name="page_2"/>
      </widget>
      <widget class="QLabel" name="lblusuarioactivo">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>10</y>
         <width>201</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Copperplate Gothic Bold</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(160, 188, 255);</string>
       </property>
       <property name="text">
        <string>Bienvenido, Usuario</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>744</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
